juspay/hyperswitch
# feat(core): add config for update_mandate_flow

## Type of Change



## Description
 add config for update_mandate_flow and allow only those payment_methods that support update while Listing PaymentMethods For Merchants


### Additional Changes





## Motivation and Context



## How did you test it?
- Create an MA and a MCA for Cybersource 
> Create an update_,mandate payment using cyber source it'll be successful
- Create  an MA and a MCA forStripe 
>  Create an update_,mandate payment using cyber source it should throw a 5xx

- Now ListPaymentMethodsForMerchnat in an Update Flow, we only list cards


## Checklist
